Lesson11_Task01.py

```diff
@@ -0,0 +1,39 @@
+"""
+Home-task 11 Generators
+
+# #############################################################################
+# write generator function that has as input some range value and chunk_size
+# produces output as lists with len == chunk size
+# Example:
+# Call:  chunk_generator(range(11), chunk_size=3) ->
+# Output:  [0, 1, 2]
+#          [3, 4, 5]
+#          [6, 7, 8]
+#          [9, 10]
+# #############################################################################
+
+"""
+
+
+def chunk_generator(data_range, chunk_size):
+    # Create a lambda function to get the next chunk
+    next_chunk = lambda x: [x + i for i in range(chunk_size)]
```

Comment:
Interesting solution, but a little bit complicated.
Please examine such code 
```
def chunks_generator(range_val, chunk_size=3):
    _range = range_val
    while _range:
        yield list(_range[:chunk_size])
        _range = _range[chunk_size:]


print(*[chunk for chunk in chunks_generator(range(23), chunk_size=5)], sep='\n')
```
loop that you have for chunk inside lambda always could be replaced with correct slice. Just pay attention to this fact. This could be helpful for you.